modules_core/search-box.js

```diff
@@ -47,19 +43,7 @@ exports.create = function (api) {
       }
     }
 
-    // delay until the element has a parent
-    setTimeout(function () {
```

Comment:
I'm not sure if it's terrible, but I just gave suggest-box what it wanted in the `build_suggest_box` function - a parent to reference.... 

It means this Timeout isn't needed ...